internal/runtime/metrics/metrics.go

```diff
@@ -58,8 +60,26 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.001, 2, 10),
 		}, []string{"host", "group", "version", "hook"}),
 	}
+	// ResponseTotal reports response results.
+	ResponseTotal = responseTotalObserver{
```

Comment:
IIRC the general best practice would be to add a `status` label to the `requests_total` metric. (not introducing a separate metric for request & response).

Reply:
(RequestsTotal also already has response "data" today, but it just includes the HTTP status code and is missing our status)

Reply:
That was my first intention - to add another label to the existing `requests_total` metric. But then 2 question arises:
1. This label is actual only if the response has status 200. What to do for other statuses?
2. This label is actual only if the response body is correctly decoded. What to do if the request returned status 200, but there was an error decoding the response body?
I was unable to come up with a solution to what to do in this cases, that's why decided to add a new metric.

Reply:
To be honest I'm not sure what the best practice here is. Might be good to do bit of research (I don't really have the time for it right now).

But I think we shouldn't have some response information on a request metric and then this part on a response metric.

Maybe it's fine to set this label to Success | Failure | Unknown?

Reply:
@sbueringer Updated my PR, added `status` to `requests_total`, so it will look like:
```
# TYPE capi_runtime_sdk_requests_total counter
capi_runtime_sdk_requests_total{code="200",group="hooks.runtime.cluster.x-k8s.io",hook="Discovery",host="capi-lifecycle-hooks-webhook.system-capi.svc:443",status="Success",version="v1alpha1"} 3
```